getnacelle/nacelle-js
# feat: add shopify-cart to next ref store

### WHY are these changes introduced?

Fixes [[ENG-7066](https://nacelle.atlassian.net/browse/ENG-7066)] 



### WHAT is this pull request doing?

- Adds shopify cart to the nuxt reference store
- Replace checkout with shopify cart checkout


### How to Test

- `git checkout ENG-7066/next-reference-store-shopify-cart `
- to the Next reference store project `.env` add the following: (reach out if you want the vars I used)
- 
```
SHOPIFY_SHOP_ID="<YOUR_SHOPIFY_ID>"
SHOPIFY_STOREFRONT_ACCESS_TOKEN="<YOUR_SHOPIFY_STOREFRONT_ACCESS_TOKEN>"
```

- `cd reference-stores/next`
- ` npm i && npm run build && npm run start` or `npm run dev`
- see if cart/checkout works
